flyway-core/src/test/java/org/flywaydb/core/internal/resolver/sql/SqlMigrationResolverSmallTest.java

```diff
@@ -15,6 +15,14 @@
  */
```

Comment:
Please revert these unnecessary formatting changes. They only make the PR harder to review. Thanks!